pom.xml

```diff
@@ -18,8 +28,16 @@
         </maven.checkstyle.plugin.configLocation>
     </properties>
 
+
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
```

Comment:
Here we can add `<version>3.3.1</version>`. 
BTW program working without this code line.